scenarios/my-scenario.js

```diff
@@ -0,0 +1,119 @@
+const enigma = require('enigma.js');
```

Comment:
Can we provide a better name for the file? `my-scenario.js` tells little about what kind of scenario it implements.
Something like `random-selection-scenario.js`?

Reply:
Changed